src/main/java/seedu/address/model/ModelManager.java

```diff
@@ -47,6 +48,8 @@
     private final FilteredList<ReadOnlyPerson> filteredPersons;
     private final FilteredList<ReadOnlyTask> filteredTasks;
 
+    private MainWindow mainWindow;
```

Comment:
Does this violate the architecture? I suggest raising a FontSizeChangedEvent instead, so that you don't need to put the UI into the model

Reply:
True... I will work on the event handling that we talked about this morning